src/Core/Prefab/GlobeView.js

```diff
@@ -363,8 +365,8 @@ GlobeView.prototype.removeLayer = function removeImageryLayer(layerId) {
 };
 
 GlobeView.prototype.selectNodeAt = function selectNodeAt(mouse) {
-    // update the picking ray with the camera and mouse position
-    const selectedId = this.screenCoordsToNodeId(mouse);
+    const picked = this.wgs84TileLayer.pickObjectsAt(this, mouse);
+    const selectedId = picked.length ? picked[0].object.id : undefined;
 
     for (const n of this.wgs84TileLayer.level0Nodes) {
```

Comment:
it is necessary to browse the tree so we have the node in `picked [0] .object`?

Reply:
I'm not sure to understand what you mean.
But we have to browse the tree to select/deselect tiles based on the picking result.

Reply:
in other PR: it will later be necessary to keep the selection to avoid traverse the tree to update the nodes